addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/TCPServerBridgeHandler.java

```diff
@@ -185,9 +185,9 @@ public void closeConnection() {
             setConnected(false);
             logger.debug("closeConnection(): Closed TCP Connection!");
         } catch (IOException ioException) {
-            logger.error("closeConnection(): Unable to close connection - {}" + ioException.getMessage());
+            logger.error("closeConnection(): Unable to close connection - {}", ioException.getMessage());
         } catch (Exception exception) {
-            logger.error("closeConnection(): Error closing connection - {}" + exception.getMessage());
+            logger.error("closeConnection(): Error closing connection - {}", exception.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace